target/linux/ramips/image/mt7621.mk

```diff
@@ -637,6 +643,25 @@ define Device/hiwifi_hc5962
 endef
 TARGET_DEVICES += hiwifi_hc5962
 
+define Device/humax_e10
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 15936k
+  DEVICE_VENDOR := HUMAX
+  DEVICE_MODEL := E10
+  DEVICE_ALT0_VENDOR := HUMAX
+  DEVICE_ALT0_MODEL := QUANTUM E10
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | \
+	edimax-header -s CSYS -m EA03 -f 0x70000 -S 0x01100000 | pad-rootfs | \
+	check-size | append-metadata
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-kernel | append-rootfs | pad-rootfs | \
+	edimax-header -s CSYS -m EA03 -f 0x70000 -S 0x01100000 | \
+	check-size | humax-factory f013c26cf0a320fb71d03356dcb6bb63
```

Comment:
Could you rebase onto the current master and try this?:
```diff
-	check-size | humax-factory f013c26cf0a320fb71d03356dcb6bb63
+	check-size | zip upg -P f013c26cf0a320fb71d03356dcb6bb63
```

Reply:
It doesn't work.

```text
/home/choryu/openwrt/staging_dir/host/bin/mkedimaximg -i /home/choryu/openwrt/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/tmp/openwrt-ramips-mt7621-humax_e10-squashfs-factory.bin -o /home/choryu/openwrt/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/tmp/openwrt-ramips-mt7621-humax_e10-squashfs-factory.bin.new -s CSYS -m EA03 -f 0x70000 -S 0x01100000
mkdir /home/choryu/openwrt/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/tmp/openwrt-ramips-mt7621-humax_e10-squashfs-factory.bin.tmp
mv /home/choryu/openwrt/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/tmp/openwrt-ramips-mt7621-humax_e10-squashfs-factory.bin /home/choryu/openwrt/build_dir/target-mipsel_24kc_musl/linux-ramips_mt7621/tmp/openwrt-ramips-mt7621-humax_e10-squashfs-factory.bin.tmp/upg -P f013c26cf0a320fb71d03356dcb6bb63
mv: invalid option -- 'P'
Try 'mv --help' for more information.
```

Reply:
Ok, you make a new patch! I'll try again!

Reply:
I tested it now, looks good! Applied!